careers/blocks/tabs/tabs.js

```diff
@@ -1,3 +1,5 @@
+import { createTag } from '../../scripts/scripts.js';
```

Comment:
Not only CSS simplifications, but still relates to https://github.com/hlxsites/caesars-careers/issues/52